plugins/talk-plugin-featured-comments/client/components/FeaturedComments.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styles from './styles.css';
+import FeaturedComment from './FeaturedComment';
+
+const isFeatured = (tags) => !!tags.filter((t) => t.tag.name === 'FEATURED').length;
+
+class FeaturedComments extends React.Component {
+  render() {
+
+    const {view : comments}  = this.props;
+
+    const featuredComments = comments.reduce((acc, curr) => {
+      if (isFeatured(curr.tags)) {
+        acc.push(curr);
+      }
+
+      return acc;
+    }, []);
```

Comment:
This is one of them :) It was just here for demo purpuses, we are not going to keep this one.